services/slackServices/users.js

```diff
@@ -0,0 +1,29 @@
+const Organization = require('../../data/dbModels/organizations');
+const User = require('../../data/dbModels/users');
+const { slackModel } = require('../../data/slackModels/slack');
+
+async function getUser(userSlackId, orgId, accessToken) {
+  try {
+    const userOtherInfo = await slackModel.user.getUser(userSlackId, accessToken);
+    const name = userOtherInfo.user.profile.real_name;
+    const avatar = userOtherInfo.user.profile.image_512;
+
+    let user = await User.readBySlackId(userSlackId);
+    let { id } = await Organization.read(orgId);
+    if (!user) {
+      const userData = {
+        org_id: id,
+        slack_mem_id: userSlackId,
+        name,
+        avatar,
+      };
+      user = await User.create(userData);
+      return user;
+    }
+    return user;
+  } catch (err) {
+    console.log(err);
```

Comment:
You may have forgotten to remove the `console.log(err);`

Reply:
I have changed this to return null. It's not a part of any HTTP request so I'm not what to do with it as the other slack services follow the same pattern. Let me know how I should proceed.